partridge/cli.py

```diff
@@ -0,0 +1,136 @@
+from collections import defaultdict
+
+import click
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+
+from .config import default_config
+from .gtfs import feed as mkfeed
+from .hmm import (
+    CandidateCache,
+    ObservationCache,
+    MeasureCache,
+    NoPathError,
+    _shape_dist_traveled,
```

Comment:
Minor: Why is the `_*` named method being exported/imported. I've always followed the pattern of `_*` methods being restricted to a single file.